tests/src/Unit/FilterFormatManagerTest.php

```diff
@@ -0,0 +1,412 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_migration\Unit;
+
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\oe_migration\FilterFormatManager;
+use Drupal\filter\Entity\FilterFormat;
+use Drupal\filter\FilterPluginCollection;
+use Drupal\filter\Plugin\FilterInterface;
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * @coversDefaultClass \Drupal\oe_migration\FilterFormatManager
+ *
+ * @group oe_migration
+ */
+class FilterFormatManagerTest extends UnitTestCase {
```

Comment:
This test looks very nice. However, consider for the future for things like this if Kernel is not easier/faster to write, and not that slow to run either. The scope of what you are testing is quite narrow so Kernel would work for this well and you'd be able to write this much faster.

That being said, you can leave this here, it's fine, it's a comment for the future to ease the work and not have to mock so many things.

Reply:
Thanks for that, Unit tests, in some cases are overkill. We already have Unit tests for all of our code but for the next PR's we'll try to convert them, if there is too muck mocking to Kernel, it'll be a nice exercise.

Reply:
If you have unit tests, don't covert them, It's fine. For new tests, it may be faster to go with Kernel tests. You don't have to mock anything with Kernel tests :)